mindsdb_sdk/connectors/rest_api.py

```diff
@@ -24,19 +24,25 @@ def wrapper(self, *args, **kwargs):
 
 
 class RestAPI:
-    def __init__(self, url=None, email=None, password=None):
+    def __init__(self, url=None, email=None, password=None, is_managed=False):
 
         self.url = url
         self.email = email
         self.password = password
+        self.is_managed = is_managed
         self.session = requests.Session()
 
         if email is not None:
             self.login()
 
     def login(self):
-        url = self.url + '/cloud/login'
-        json = {'email': self.email, 'password': self.password}
+        login_endpoint = '/api/login' if self.is_managed else '/cloud/login'
```

Comment:
We can do: try one endpoint and if not_found - use another

Reply:
Yes, it takes a little time, but
- user is not waiting of timeout, response (not_allowed or not_found) returns immediately
- ideally it happens once at connection to the server

Use case can be: 
If user put is_managed=True  - use '/api/login'
Otherwise:
- try use '/cloud/login'.
- if it returns not_allowed - try use '/api/login'